device-registry/src/main/scala/org/genivi/sota/device_registry/db/InstalledPackages.scala

```diff
@@ -53,6 +53,9 @@ object InstalledPackages {
       installedPackages ++= packages.map(InstalledPackage(device, _, Instant.now()))
     ).transactionally
 
+  def addRows(rows: Seq[InstalledPackage])(implicit ec: ExecutionContext): DBIO[Seq[Int]] =
+    DBIO.sequence(rows.map(installedPackages += _)).transactionally
```

Comment:
This whole migration should be idempotent, I think this prevents the migration from being idempotent, it will just add more and more rows, or will just fail because we are trying to add duplicate keys.